staging/src/k8s.io/component-helpers/network/utils/iptables/OWNERS

```diff
@@ -0,0 +1,14 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
This file can be a copy of https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/OWNERS instead (actually it's better to copy it, and we end by using the proper aliases)